redisvl/extensions/llmcache/semantic.py

```diff
@@ -333,19 +366,66 @@ def store(
         self._check_vector_dims(vector)
 
         # Construct semantic cache payload
+        now = time()
         id_field = self.entry_id_field_name
         payload = {
             id_field: self.hash_input(prompt),
             self.prompt_field_name: prompt,
             self.response_field_name: response,
             self.vector_field_name: array_to_buffer(vector),
+            self.inserted_at_field_name: now,
+            self.updated_at_field_name: now,
         }
         if metadata is not None:
             if not isinstance(metadata, dict):
                 raise TypeError("If specified, cached metadata must be a dictionary.")
             # Serialize the metadata dict and add to cache payload
             payload[self.metadata_field_name] = self.serialize(metadata)
```

Comment:
Ditto, I think we can move serialization to/from JSON string to utils so we can use the same technique and unit test the module.